xmodule/modulestore/xml_importer.py

```diff
@@ -900,10 +900,7 @@ def _convert_ref_fields_to_new_namespace(reference):
             try:
                 # Update library content block's children on draft branch
                 with store.branch_setting(branch_setting=ModuleStoreEnum.Branch.draft_preferred):
-                    LibraryToolsService(store, user_id).trigger_update_children_task(
-                        block,
-                        version=block.source_library_version,
-                    )
+                    LibraryToolsService(store, user_id).trigger_refresh_children(block)
```

Comment:
version argument no longer exists. It was always passed in as block.source_library_version, and it was ignored in V2.